### PR TITLE
Update README now the js-sdk has a transpile step

### PR DESCRIPTION
**Pull request by dbkr**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -58,8 +58,11 @@ to build.
 1. Install the prerequisites: `npm install`
 1. If you are using the `develop` branch of vector-web, you will probably need
    to rebuild one of the dependencies, due to
```
Comment (richvdh) [inline README.md]: s/one/some

File: README.md [hunk 2]
```diff
@@ -241,10 +244,10 @@ Finally, build and start Riot itself:
    disables caching, so do NOT use it in production.
 1. Open http://127.0.0.1:8080/ in your browser to see your newly built Riot.
 
-When you make changes to `matrix-react-sdk`, you will need to run `npm run
-build` in the relevant directory. You can do this automatically by instead
-running `npm start` in the directory, to start a development builder which
-will watch for changes to the files and rebuild automatically.
+When you make changes to `matrix-react-sdk` or `matrix-js-sdk`, you will need
+to run `npm run build` in the relevant directory. You can do this automatically
```
Comment (richvdh) [inline README.md]: Annoyingly, I think it is `npm run compile` for the js-sdk.

Comment (richvdh) [inline README.md]: It would obviously be preferable to fix this by bringing the js-sdk into line.

File: README.md [hunk 3]
```diff
@@ -58,8 +58,11 @@ to build.
 1. Install the prerequisites: `npm install`
 1. If you are using the `develop` branch of vector-web, you will probably need
    to rebuild one of the dependencies, due to
-   https://github.com/npm/npm/issues/3055: `(cd node_modules/matrix-react-sdk
-   && npm install)`
+   https://github.com/npm/npm/issues/3055:
+   ```
```
Comment (richvdh) [inline README.md]: the formatting is broken here: https://github.com/vector-im/riot-web/blob/5258bf9c8455e97ce59e82150c51e31c0c681a44/README.md#building-from-source

File: README.md (hunk 2 above)
Comment (dbkr) [inline README.md]: Fixed in https://github.com/matrix-org/matrix-js-sdk/pull/320 by having npm run build do the things you need (and more!)

**Comment by dbkr** [comment]
ptal

**Pull request merged**